autoinstall/FreeBSD/installerconfig

```diff
@@ -0,0 +1,102 @@
+
+debug=yes
+nonInteractive=yes
+
+PARTITIONS="da0 gpt"
+DISTRIBUTIONS="kernel.txz base.txz kernel-dbg.txz lib32.txz src.txz ports.txz"
+
+#!/bin/sh
+# Set hostname
+sysrc hostname="FreeBSD"
+
+gpart bootcode -b /boot/pmbr -p /boot/gptboot -i 1 da0
+# Set Time Zone to UTC
+echo "Setting Time Zone to UTC ..."
+/bin/cp /usr/share/zoneinfo/UTC /etc/localtime
+/usr/bin/touch /etc/wall_cmos_clock
+/sbin/adjkerntz -a
+
+echo "Set DHCP to NIC nic0 ..." > /dev/ttyu0
+ifdev=$(ifconfig | grep '^[a-z]' | cut -d: -f1 | head -n 1)
+echo "Get ifname ${ifdev}"
+sysrc ifconfig_${ifdev}=DHCP
+
+# Get DHCP for nic0
+echo "Get IP with dhclient ..." > /dev/ttyu0
+dhclient ${ifdev}
+sleep 15
+echo "Check network ..." > /dev/ttyu0
+ifconfig > /dev/ttyu0
+
+# Set Proxy. without the proxy, maybe it will take more time to download pakages xorg & gnome
+{% if use_localhost_proxy is defined and use_localhost_proxy %}
+setenv HTTP_PROXY {{ http_proxy_localhost }}
+{% endif %}
+
+# Installing packages
+echo "Installing packages ..." > /dev/ttyu0
+env ASSUME_ALWAYS_YES=YES pkg bootstrap -y
+env ASSUME_ALWAYS_YES=YES pkg update -f
+env ASSUME_ALWAYS_YES=YES pkg upgrade -f
```

Comment:
Keep "env ASSUME_ALWAYS_YES=YES pkg update -f" because we need to update repo before install packages.
Delete "env ASSUME_ALWAYS_YES=YES pkg upgrade -f"

Thanks